test/client/components/script/script.test.js

```diff
@@ -0,0 +1,79 @@
+import { fetchScript, fetchScriptQuestions } from '../../../../public/src/actions/script';
+import fixtures from '../../client_fixtures';
+import { checkObjectProps } from '../../client_test_helpers.js';
+
+const { scriptQuestionsFixture: questions,
+        admin_scripts: scripts } = fixtures.scriptViewFixtures;
+
+const props = {
+  id: 1,
+  scripts,
```

Comment:
Why is scripts here? Isn't this just for one script and its associated questions?